### ISSUE TITLE
New User Guide

### ISSUE THREAD

**Issue by apainintheneck** [apainintheneck/SQL-Study#38]
During user testing the tester mentioned that they think it would be nice if there was some sort of user guide that tells people how to navigate the website. Do you guys think that would be a good idea? Where would we put it? On the dashboard page?

**Comment by apainintheneck** [comment]
I think in theory we could have it on the dashboard page instead of the links to the different content sections since those links are already available in the sidebar.

**Comment by brycerandolph** [comment]
Yeah I agree completely, this was talked about during my user test as well. The dashboard seems like a good place.

**Comment by jeanniedavis** [comment]
i can work on this, so just to confirm turn the dashboard to a "user guide" on how to navigate the site?

**Comment by brycerandolph** [comment]
I think he means replacing the section I circled here with a user guide.
![image](https://user-images.githubusercontent.com/33710291/144155303-b57fd706-5f62-4425-b2d8-cf425421b1fa.png)

**Comment by apainintheneck** [comment]
Yep, that’s what I had in mind but, of course, I’m always open to other ideas.
